near-abi-client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "near-abi-client"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/near/near-abi-client-rs"
+
+[dependencies]
+anyhow = "1.0"
+convert_case = "0.5"
+near-sdk = { git = "https://github.com/near/near-sdk-rs.git", rev = "9ab6df8ed35e67c247299059d8277c2170a0a91b", features = ["abi"] }
```

Comment:
note, you might want to wait until the 4.1 pre-release is released and update this to avoid the git dep in a few places here